scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -1,3 +1,10 @@
+//VERSION 2.0
+//  Modified by Alezia Kurdis on on 02/27/2020
+//  for "Project Athena"
+//  
+//  Addition of a tab mechanism instead of collapsible sections to reduce the scrolling.
+//
+//VERSION 1.0
```

Comment:
Same comment per entityProperties.html.

Reply:
Changes for this will be committed in PR #215 related to properties changes to avoid regression.